parlai/mturk/tasks/light/light_chat_eval/worlds.py

```diff
@@ -0,0 +1,372 @@
+        self.mturk_agent.observe(self.block_act)
+        self.mturk_agent.mturk_manager.soft_block_worker(
+            self.mturk_agent.worker_id)
+        act = self.mturk_agent.act()
+        while not is_disconnected(act):
+            self.mturk_agent.observe(self.block_act)
+            act = self.mturk_agent.act()
+        return True
+
+    def __init__(self, opt, mturk_agent):
+        self.mturk_agent = mturk_agent
+        self.opt = opt
+        self.did_complete = False
+        self.wrong = 0
+        self.episodeDone = False
+
+    def parley(self):
+        self.mturk_agent.update_agent_id('TestEmote')
+        first_act = {
+            'id': 'System',
+            'text': 'FIRST_TURN',
+            'task_data': {
+                'wrong': 0,
+                'turn': 'FIRST_TURN',
+                'actions': self.GESTURES,
+                'agent_id': 'Guard',
+                'text':
+                    'Bahahaha that\'s a great one! Where\'d you get that from?',
+                'persona':
+                    'I\'m a guard of the royal family. I have a loud laugh, '
+                    'and people hear it often as I love jokes. I stand up for '
+                    'rightousness, and have a short temper when it comes to '
+                    'insults against the king. Sometimes you need to knock '
+                    'some sense into people.',
+                'base_name': 'Guard',
+                'partner_name': 'Jester',
+                'setting':
+                    'You are in the servants\' quarters. Many people are '
+                    'sitting around waiting to be called for services. It\'s '
+                    'cozy, but not cramped. A chest is here. A Jester is here. '
+                    'You are carrying a spear.',
+            }
+        }
+        self.mturk_agent.observe(first_act)
+        act = self.mturk_agent.act()
+        if is_disconnected(act):
+            self.episodeDone = True
+            return
+        while act['text'] != 'gesture laugh':
+            self.wrong += 1
+            if self.wrong > 3:
+                return self.block_loop()
+            first_act['task_data']['wrong'] = self.wrong
+            self.mturk_agent.observe(first_act)
+            act = self.mturk_agent.act()
+            if is_disconnected(act):
+                self.episodeDone = True
+                return
+        self.mturk_agent.update_agent_id('TestSpeech')
+        correct_phrase = (
+            'Now you better watch your tongue Jester. '
+            'I won\'t have you badmouthing our king.'
+        )
+        second_act = {
+            'id': 'System',
+            'text': 'SECOND_TURN',
+            'task_data': {
+                'wrong': 0,
+                'turn': 'SECOND_TURN',
+                'curr_message_context': {'action': 'gesture frown'},
+                'actions': [
+                    'You think you can say whatever you want '
```

Comment:
the mix of newlines with and without , is rather jarring. Should we just override line length?